kuzzleio/kuzzle-admin-console
# Csv export

## What does this PR do ?
#787 

Allow user to export a csv file of the current column view (only the current page with the selected columns and the search query)

### How should this be manually tested?
  - Step 1 : create a testIndex index, a testCollection collection, some documents with
  - Step 2 : Check the column view and select some property to display
  - Step 3 : Click on the export button
  ...

### Other changes
/

### Boyscout
/
### Screenshots (if appropriate)
/
